src/index.css

```diff
@@ -51,6 +52,38 @@ code {
   margin-right: 5%;
   margin-left: 5%;
 }
+
+.gameDescription {
+  width: auto;
+  margin-top: 10%;
+  text-align: center;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+
+.gameDescription h3 {
+  color: #00243f;
+  font-weight: 600;
+}
+
+.gameDescription h5 {
+  margin-top: 5%;
+  color: #00243f;
+  text-transform: uppercase;
+  font-weight: 700;
+  font-size: 1.5em;
+  text-shadow: 2px 2px #abb3bc;
+}
+
+.landingPageMenu {
+  margin-top: 5%;
+  width: 30vw;
```

Comment:
Will flex be a better option here? What would happen on higher res screens?

Reply:
Yep will make that change